resources/css/style.css

```diff
@@ -656,6 +658,72 @@ footer p {font-size: 12px;}
 
   body.checkout label span.custom {float: left;}
 
+  /* Credit Card Form */
+  .ls-card-form .card-form-input:focus {
+    background: #ffffff;
+  }
+  .ls-card-form .card-form-label {
+    font-size: 12px;
+    letter-spacing: 0px;
+    font-weight: 500;
+    margin-left: 0px;
+    padding-bottom: 0px;
+    margin-bottom: 3px;
+  }
+  .ls-card-form .card-form-input {
+    background: #ffffff;
+    border-radius: 0px;
+    box-shadow: inset 0 1px 3px 0 rgba(0, 0, 0, 0);
+    height: 35px;
+    font-family: Courier New, Courier, monospace;
+    letter-spacing: 0px; 
+    font-weight: 400;
+    font-size: 16px;
+    color: rgb(58, 61, 63);
+    line-height: 32px;
+  }
+  .ls-card-form .card-form-input::-webkit-input-placeholder {
+    font-family: Courier New, Courier, monospace;
+    color: rgb(58, 61, 63) !important;
+    font-weight: 500;
+    font-size: 16px;
+    letter-spacing: 0px;
+  }
+  #submit-button, .ls-card-form .delete-button {

```

Comment:
CSS lint guidelines require that each selector gets its own line:

`#submit-button, .ls-card-form .delete-button {`

Should be:

```
#submit-button,
.ls-card-form .delete-button {
```